src/ui/animation.js

```diff
@@ -0,0 +1,142 @@
+import { CONSTANTS } from "../constants/constants.js";
+import { characterData } from "../cookie-io.js";
+import { capitalize } from "../utilities.js";
+
+export class Animation extends Phaser.Scene {
+    constructor() {
+        super({ key: CONSTANTS.SCENES.ANIMATION });
+    }
+
+    preload() {
+        // Load icons for all skills
+        for (const skill in characterData.getSkills()) {
+            this.load.image(
+                skill + "Icon",
+                "src/assets/ui/icons/" + capitalize(skill) + ".png"
+            );
+        }
+    }
+
+    showXp(skill, xp) {
+        if (xp <= 0) {
+            return;
+        }
+
+        const startX = 470;
+        const startY = Math.floor(this.cameras.main.height / 2);
+        const endX = startX;
+        const endY = 20;
+
+        // Gold runescape font
+        const font = {
+            font: "24px runescape",
```

Comment:
I think it looks better with font size of 18px and the startY value at 100

Reply:
Done here: https://github.com/EthanVieira/rs-clicker/pull/65/commits/f1e491a0b907e0df873ae34268d4d6cac09031f5